services/ui/ws/window_tree.h

```diff
@@ -164,6 +164,10 @@ class WindowTree : public mojom::WindowTree,
             window));
   }
   WindowManagerState* window_manager_state() {
+    if (!external_mode_wm_states_.empty()) {
+      WindowManagerState* wms = external_mode_wm_states_.begin()->get();
```

Comment:
this will return the WindowManagerState of the first ws::Display created always. This is what we want?

Reply:
not needed anymore.